Stylesheets/Components/badge.css

```diff
@@ -0,0 +1,81 @@
+.badge{
+    position: relative;
+    margin:1rem;
+ 
+}
+
+.badge i{
+    font-size: 2rem;
+    position: relative;
+    display: inline-block;
+ 
```

Comment:
extra space here as well

Reply:
Thanks vanshita will use prettier before pushing the code.